### PR TITLE
Fix dynamic for freecommander.com

### PR DESCRIPTION
**Pull request by Gitoffthelawn**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4647,6 +4647,20 @@ img[src$="layout/logo.svg"]
...
+freecommander.com
+
+CSS
+.am-body,
+#subsilver-nav-topic {
+    background-image: none !important;
+}
+.header > .row-item,
+.body-blok-h3 h3 {
+    background: #000 !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4659]:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

**Comment by Gitoffthelawn** [comment]
@gusted It doesn't look like this made it into DR v4.9.35

What needs to be done to get it included in the next release?

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:4659]: I thought this had to be done?

Comment (Gitoffthelawn) [reply src/config/dynamic-theme-fixes.config]: That's weird... it wasn't showing that I needed to apply the change you made.  I did it now, but the original source is now updated... will it still merge ok?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Yup, seems to be fine. Github is sometimes weird, not the first time someone didn't got a email or the correct UI for changes.

**Review by Gusted — APPROVED** [review]

**Comment by Gitoffthelawn** [comment]
@Gusted Because of the bug that wasn't saving DR devtools changes, I wasn't able to test your edit for this PR.

Unfortunately, the edit you made broke it.

See the "Statistics" row on:
https://freecommander.com/forum/

What do you propose?

**Comment by Gusted** [comment]
> What do you propose?

What about:
```css
.body-blok-h3 h3 {
    background-image: none !important;
}
```

**Pull request merged**